stories/StepsIndicator/Docs.mdx

```diff
@@ -51,48 +46,28 @@ The attributes that can be passed through are listed below.
   </tbody>
 </table>
 
-An array of length `length` contains `<Step>` components, the colour of which is controlled by the `selected` data attribute (`true | false`). By default we use `selectedIndex` to point to the component that requires this attribute to be true. For higher customisability, we also include the `className` and `childClassName`. These respectively refer to the `<div>` containing all the Steps, and the `<Step/>` themselves.
-
-For example, if you wanted the border of the `<div>` container to be grey, and the colour of the selected dot to be a large red:
+An array of length `length` contains `<Step>` components, the colour of which is controlled by the `selected` data attribute (`true | false`). We use `selectedIndex` to point to the component that requires this attribute to be true.
 
-<Canvas of={StepsStories.Customised} />
+The `onClick` function has access to the index of the `<Step>` component that was just clicked and can use that to update the props passed to `<StepIndicator>`.
 
-## Step
+## Example usage
 
 ```ts
-import ProgressDefault from '@/assets/build/progressdefault.icon'
-import { cn } from '@/lib/utils'
-import { cva } from 'class-variance-authority'
-
-interface StepProps {
-  idx?: string
-  selected: 'true' | 'false'
-  className?: string
-}
-
-const stepVariant = cva([
-  'text-inherit',
-  'fill-current',
-  'data-[selected=true]:text-primary-700',
-  'dark:data-[selected=true]:text-white'
-])
+export const StepsIndicatorDemo: React.FC = () => {
+  const [selectedIndex, setSelectedIndex] = useState<number>(1)
+  const onClick = (i: number) => setSelectedIndex(i)
 
-const Step = ({ idx, selected, className }: StepProps) => {
   return (
-    <ProgressDefault
-      key={idx}
-      data-selected={selected}
-      className={cn(stepVariant(), className)}
+    <StepsIndicator
+      length={10}
+      selectedIndex={selectedIndex}
+      onClick={onClick}
     />
   )
 }
-
-export { Step, StepProps }
-
-
```

Comment:
The two examples were the same. I replaced one with the example component I set up to allow the selected step to change.